tv-shows-app/src/app/(auth)/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import { Container, Flex } from "@chakra-ui/react";
+import { AuthRedirect } from "@/components/shared/AuthRedirect/AuthRedirect";
+
+export default function AuthLayout({
+	children,
+}: Readonly<{
+	children: React.ReactNode;
+}>) {
+	return (
+		<AuthRedirect>
```

Comment:
Mislim da smo pričali o ovome, ovo je good enough ali nema razloga zašto ne staviti `AuthRedirect` na svaki page (na kojem nam treba) zasebno.
Ne moraš "ispravljati" ako ne stigneš, ali čisto da ne mislis da je to losa praksa.